
# HG changeset patch
# User Peter Bloomfield <peterbloomfield@bellsouth.net>
# Date 1313373015 14400
# Node ID 1f70e6de755c13c5c2e555935725c1dafd258d6a
# Parent  56a41f3309b4c6b9a2ffed4c1ac19597ab6c1177
Use gtk_bin_get_child instead of accessing the struct member directly

This is the only code change needed to compile with GTK3.  There are some build
changes needed too, but those will happen in a soon-to-be-created branch.

diff -r 56a41f3309b4 -r 1f70e6de755c gtkspell/gtkspell.c
--- a/gtkspell/gtkspell.c	Sun Aug 14 21:32:46 2011 -0400
+++ b/gtkspell/gtkspell.c	Sun Aug 14 21:50:15 2011 -0400
@@ -310,7 +310,7 @@
 
 	get_word_extents_from_mark(spell->buffer, &start, &end, spell->mark_click);
 	oldword = gtk_text_buffer_get_text(spell->buffer, &start, &end, FALSE);
-	newword = gtk_label_get_text(GTK_LABEL(GTK_BIN(menuitem)->child));
+	newword = gtk_label_get_text(GTK_LABEL(gtk_bin_get_child(GTK_BIN(menuitem))));
 
 	if (debug) {
 		g_print("old word: '%s'\n", oldword);

